src/components/StyleSelector.jsx

```diff
@@ -212,12 +268,44 @@ const StyleSelector = (props) => {
               anchorOrigin={{vertical: 'bottom', horizontal: 'center'}}
               transformOrigin={{vertical: 'top', horizontal: 'center'}}
               >{availableSkus}</Menu>
-            </div>
+            </Grid>
             :
-            <span></span>
+            <Grid item xs={4}></Grid>
           }
-        </div>
-      </div>
+        </Grid>
+        <Grid item xs={12}>
+          <Grid item xs={12} className={classes.buttonWrappers}>
+            <LightTooltip title={warning === '' ? '' : warning} placement="left">
```

Comment:
This seems a little redundant.

Reply:
I wish it were, but getting the grid to layout correctly takes a certain amount of wrapping. Thanks for the sharp eye tho.